src/index.css

```diff
@@ -18,14 +19,16 @@ h1 {
 }
 
 .art-image {
-  width: 398px;
-  height: 299px;
+  width: 100%;
+  height: auto;
+  aspect-ratio: 4 / 3;
+  object-fit: cover;
 }
```

Comment:
In the designs the image was 398px:299px - i allowed myself to just round it up and keep the aspect ratio 4:3.

Let me know if you object